fix adaption typo (#1886)
d2l-ai/d2l-en: chapter_attention-mechanisms/attention-scoring-functions.md

    # Shape of `valid_lens`: (`batch_size`,) or (`batch_size`, no. of queries)
    def forward(self, queries, keys, values, valid_lens=None):
        d = queries.shape[-1]

>         # Set `keys.transpose(1,2)` to swap the last two dimensions of `keys`

Reviewer: ```suggestion
        # Swap the last two dimensions of `keys` with `keys.transpose(1,2)`
```
